tango/pyutil.py

```diff
@@ -296,21 +296,21 @@ class and its constructor is not public. Example::
     def document_method(method_name, desc, append=True):
         return __document_method(Util, method_name, desc, append)
 
-    #    def document_static_method(method_name, desc, append=True):
-    #        return __document_static_method(_Util, method_name, desc, append)
-
-    #    document_static_method("instance", """
-    #    instance(exit = True) -> Util
-    #
-    #            Static method that gets the singleton object reference.
-    #            If the class has not been initialised with it's init method,
-    #            this method prints a message and aborts the device server process
-    #
-    #        Parameters :
-    #            - exit : (bool)
-    #
-    #        Return     : (Util) the tango Util object
-    #    """ )
+    def document_static_method(method_name, desc, append=True):
+        return __document_static_method(Util, method_name, desc, append)
+
+    document_static_method("instance", """
+    instance(exit = True) -> Util
+
+           Static method that gets the singleton object reference.
+           If the class has not been initialised with it's init method,
+           this method prints a message and aborts the device server process
+
+       Parameters :
+           - exit : (bool)
```

Comment:
Would be useful to add more detail somewhere here.  E.g., it raises a DevFailed exception instead of aborting if `exit` is set to `False`.